### ISSUE TITLE
[IBEX] Integrate IBEX account creation into Galoy account creation - Bounty #2

### ISSUE THREAD

**Issue by islandbitcoin** [lnflash/flash#7]
This bounty can be claimed by completing and testing the following tasks:

- [ ] Call the IBEX Create Account API upon Device account creation in galoy. sample code [here](https://github.com/lnflash/flash/blob/8d9259feb95d8cc745f201755f8c70f50d90064b/src/servers/authorization/device-account.ts#L63)
- [ ] Save the IBEX API response (specifically the `id` field) as the galoy USD walletId (instead of galoy generating an ID by default)
- [ ] Test to confirm that the USD walletId and the IBEX accountId are the same in the GraphQL playground (localhost:4002/graphql)

This should take less than 3 hours to complete. Bounty Details are [here](https://community.sphinx.chat/org/tickets/ck8v55o4nncjnaefo080?owner_id=02c87de4005ec7d6256be1a018c9693ab18ec595d87cded5fdcff72c86466059b6&created=1695820982)

**Comment by kevkevinpal** [comment]
are you working off a specific branch for the sample code?

**Comment by islandbitcoin** [comment]
> are you working off a specific branch for the sample code?

yes, please see the link in the issue. The brach is https://github.com/lnflash/flash/tree/review

**Comment by kevkevinpal** [comment]
@islandbitcoin I opened a draft PR but I guess I'm kinda stuck on how to test, is there a wallet or sample API request I can use for account creation?

**Comment by islandbitcoin** [comment]
good question! I was trying to add those instructions to the bounty but lost the post. 

You need to download and run [ibex-plugin](https://github.com/lnflash/ibex-plugin) locally, and that will give you the API calls to test. The postman.json file in there should have all the test credentials you need to run tests.

**Comment by islandbitcoin** [comment]
I'm working on updating the ibex-plugin so that you can run it just by typing `docker run -d -p 8760:4000 dreadnterrible/ibex-plugin:latest` 

please hold... your PR is important to us... :)

**Comment by islandbitcoin** [comment]
ok got it up on docker hub. run 
```
docker run -d -p 8760:4000 dreadnterrible/ibex-plugin:latest

```

and you can use `localhost:8760/api/v1/account` to test

**Comment by islandbitcoin** [comment]
you put the createAccountforDeviceAccount function with your code, but its already in 'login.ts'. isn't this gonna duplicate the account creation?

**Comment by ecurrencyhodler** [comment]
Closed by #9
